### PR TITLE
doc(hansbug): add c51's benchmark

### PR DESCRIPTION
**Pull request by HansBug**
Add benchmark for c51 algorithm

### CODE REVIEW

File: source/hands_on/c51_qrdqn_iqn.rst [hunk 1]
```diff
@@ -61,9 +61,38 @@ The network interface C51 used is defined as follows:
...
+   * - environment
+     - best mean reward
+     - evaluation results
+     - config link
+     - comparison
+   * - | Pong
+       | (PongNoFrameskip-v4)
+     - 20.6
+     - .. image:: images/benchmark/c51_pong.png
+     - `config_link_p <https://github.com/opendilab/DI-engine/blob/09050dba6b569b47693f483e8c1e53f836a51d74/dizoo/atari/config/serial/pong/pong_c51_config.py>`_
```
Comment (PaParaZz1) [inline source/hands_on/c51_qrdqn_iqn.rst:81]: use main branch config link

Comment (HansBug) [reply source/hands_on/c51_qrdqn_iqn.rst]: Completed!

**Comment by LuciusMos** [comment]
image
pseudo code部分公式格式有误

**Comment by HansBug** [comment]
> image pseudo code部分公式格式有误

Fixed, PTAL 😺 .

**Pull request merged**